kofta/src/vscode-webview/pages/Home.tsx

```diff
@@ -70,19 +72,28 @@ export const Home: React.FC<HomeProps> = () => {
             />
           </div>
         ) : null}
-        {rooms.map((r) =>
+        {rooms.map(r =>
           r.id === currentRoom?.id ? null : (
             <div className={tw(`mt-4`)} key={r.id}>
               <RoomCard
                 onClick={() => {
                   wsend({ op: "join_room", d: { roomId: r.id } });
+                  // Show joined message in chat
+                  setTimeout(() => {
```

Comment:
don't send this from the client-side, I'll trigger it from the backend

Reply:
Yeah O kay I will remove it now